sky/packages/sky/lib/base/lerp.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:sky';
```

Comment:
The problem is that they handle `null`.  If we move them into the class, the `this` object can't be null.